config/envs/default.py

```diff
@@ -44,3 +44,11 @@ class DefaultConfig(object):
         raise TypeError("ADDITIONAL_EMAIL_LOOKUPS must be a dictionary")
 
     TF_SUBMITTER_ROLE = "TF_MONITORING_RETURN_SUBMITTER"
+
+    # Gov Notify
+    NOTIFY_API_KEY = os.environ.get(
+        "NOTIFY_API_KEY", "fake_key-0ab1234a-12a3-12ab-12a3-a1b2cd3e4f5g-a123b456-1a23-1abv-a1bc-123a45678910"
```

Comment:
Will this not cause an exception for people running locally unless they have the real key? Maybe we can add a boolean to enable email notifications?

Reply:
Added the boolean instead - good shout